doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -1093,13 +1242,13 @@ When executed, the ``ERASEALL`` command performs the following operations:
 .. note::
   Page 1 of the MRAM10 NVR is preserved and not erased.
 
-To explicitly permit the ``ERASEALL`` command, disable erase protection by clearing the UICR.ERASEPROTECT field in the application's UICR.
```

Comment:
Could you fix these as well: 
Line 1082: ``ERASEALL`` commmand-> ``ERASEALL`` command
Line 939: * The sum of these to regions must be 4kB aligned. -> * The sum of these two regions must be 4kB aligned.